third_party/terraform/utils/node_config.go.erb

```diff
@@ -451,7 +449,6 @@ func expandNodeConfig(v interface{}) *containerBeta.NodeConfig {
 	return nc
 }
 
-<% unless version == 'ga' -%>
 func expandWorkloadMetadataConfig(v interface{}) *containerBeta.WorkloadMetadataConfig {
```

Comment:
I think we need to address the flatten methods for `WorkloadMetadataConfig` as well.